eth_tester/main.py

```diff
@@ -333,9 +333,17 @@ def get_transaction_receipt(self, transaction_hash):
         raw_transaction_hash = self.normalizer.normalize_inbound_transaction_hash(
             transaction_hash,
         )
+        transaction = self.get_transaction_by_hash(transaction_hash)
         raw_receipt = self.backend.get_transaction_receipt(raw_transaction_hash)
         self.validator.validate_outbound_receipt(raw_receipt)
         receipt = self.normalizer.normalize_outbound_receipt(raw_receipt)
+
+        byzantium_fork_block = None
+        if 'FORK_BYZANTIUM' in self.get_supported_forks():
+            byzantium_fork_block = self.get_fork_block('FORK_BYZANTIUM')
+        status = int.from_bytes(receipt.pop('state_root'), byteorder='big')
```

Comment:
https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionreceipt

According to the spec, we should only remove the `state_root` *if* we are putting the `status` field in, so this should probably be within the next `if` block.